app/assets/javascripts/multirange.js

```diff
@@ -0,0 +1,88 @@
+(function() {
+"use strict";
```

Comment:
indentation here is wrong. For the entire file.

Reply:
What is the meaning of this file? I know nothing

Reply:
It's the script for the double slider